rtv/__main__.py

```diff
@@ -24,7 +24,6 @@ def load_config():
     Search for a configuration file at the location ~/.rtv and attempt to load
     saved settings for things like the username and password.
     """
-
     config = configparser.ConfigParser()
```

Comment:
I would say rename this `config` to something else e.g.
```python
conf = configparser.ConfigParser()
```
That way you won't have to use a special import
```
from . import config as rtvconfig
```